Makefile

```diff
@@ -64,11 +67,13 @@ build: gluon-prepare
 	done
 
 manifest: build
-	${GLUON_MAKE} manifest
+	for branch in experimental testing stable; do \
+		${GLUON_MAKE} manifest GLUON_AUTOUPDATER_BRANCH=$$branch;\
```

Comment:
`$$branch`?

as asked in https://github.com/freifunkMUC/site-ffm/commit/864c99e88f6f2bdde1e855d6d56f3c9ec8be45c0#diff-76ed074a9305c04054cdebb9e9aad2d818052b07091de1f20cad0bbac34ffb52R71

Reply:
https://www.gnu.org/software/make/manual/html_node/Variables-in-Recipes.html

Reply:
okay :+1: